src/dhcpmon/.gitignore

```diff
@@ -0,0 +1,5 @@
+debian/*
+!debian/changelog
+!debian/compat
+!debian/control
+!debian/rules
```

Comment:
Merge them into one line rule?

Reply:
As with the above, these are inverse ignore rules. We ignore all, then un-ignore the files we want to keep. Otherwise, there would be more lines :)

Reply:
`debian/*` make things complex.

Reply:
I disagree. There are more files we want to ignore than the number we want to commit.